### ISSUE TITLE
Comment in the package creation code breaks Go to symbol/outline

### ISSUE THREAD

**Issue by MrPixel1** [zabel-xyz/plsql-language#68]
In a PL/SQL package body source file, the first code block works well. The outline shows up with the myProc procedure and Go to symbol works as expected.

```
CREATE OR REPLACE PACKAGE BODY
myPackage AS

PROCEDURE myProc IS
BEGIN
  NULL;
END;

END;
```

However, as soon as a comment is inserted before the package name, the outline/Go to symbol does not work anymore. It doesn't matter if the comment is a block comment or a single line comment, nor does the content of the comment matter (even just the two dashes alone break the functionality).

```
CREATE OR REPLACE PACKAGE BODY
-- Some comment
myPackage AS

PROCEDURE myProc IS
BEGIN
  NULL;
END;

END;
```

It seems to be a little regression. Up to version 1.4.2, it was working. Yet, in that version, the comment would lead to the symbols being recognized as definitions (not the whole implementation). This is not a big deal breaker though, since navigating through symbols worked.
After version 1.4.2 though, it wasn't working anymore.

**Comment by zabel-xyz** [comment]
Have you set setting `plsql-language.commentInSymbols` to true. (see #35)

Comment in a single line was never supported, an improvement...

**Comment by zabel-xyz** [comment]
Fix in version 1.8.0
Use settings plsql-language.commentInSymbols
